Library/Homebrew/language/node.rb

```diff
@@ -0,0 +1,32 @@
+module Language
+  module Node
+    def self.npm_cache_config
+      "cache=#{HOMEBREW_CACHE}/npm_cache\n"
+    end
+
+    def self.setup_npm_environment
+      # explicitly use our npm and node-gyp executables instead of the user
+      # managed ones in HOMEBREW_PREFIX/lib/node_modules which might be broken
+      ENV.prepend_path "PATH", "#{Formula["node"].opt_libexec}/npm/bin"
```

Comment:
`opt_libexec` returns a `Pathname` instance. Thus, a slightly more readable version of this line would be:

```ruby
ENV.prepend_path "PATH", Formula["node"].opt_libexec/"npm/bin"
```